js/main.js

```diff
@@ -6,17 +6,79 @@ function getRandomInteger(min, max) {
   }
   return Math.floor(rand);
 }
-getRandomInteger(1, 3);
 
 
 // Результат: случайное число с плавающей точкой число из диапазона "от...до"
 function getRandomFloat(min, max, fraction) {
   const rand = min + Math.random() * (max - min);
   if( max <= min ) {
-    throw new Error('getRandomInteger — максимальное число меньше или равно минимальному');
+    throw new Error('getRandomFloat — максимальное число меньше или равно минимальному');
   }
   return parseFloat(rand).toFixed(fraction);
 }
 
-getRandomFloat(1.1, 2.3, 2);
+const TYPE_HOUSE = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel',
+];
+
+const CHECKIN_CHECKOUT = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const FACILITIES = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+  'conditioner',
+];
+
+const PHOTOS = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const greatFacilities = new Array(getRandomInteger(0, 5)).fill(null).map(() => FACILITIES);
+
+const SIMILAR_WIZARD_COUNT = 10;
+
+const getRandomArrayElement = (elements) => {
+  return elements[getRandomInteger(0, elements.length - 1)];
+};
```

Comment:
Тут и в других местах есть ошибка линтера: `Unexpected block statement surrounding arrow body; move the returned value immediately after the '=>'`

Она означает, что в функциях, которые сразу возвращают значение нужно убирать return и писать возвращаемое значение сразу после стрелки:
```suggestion
const getRandomArrayElement = (arr) => arr[getRandomInteger(0, arr.length - 1)];
```